### ISSUE TITLE
Bug: Network Drives option not working

### ISSUE THREAD

**Issue by jaseinatl** [files-community/Files#11951]
### Description

The only thing I see in the Network Drives sidepanel is a folder called Network. Even clicking on the Network folder shows the "busy" cursor but never displays any content.

I was expecting to see two types of options perhaps separated by a line. The top half would be all of the PCs on my network and mayube all of the network devices total. Then the second half would give me all of my mapped locations

### Steps To Reproduce

Launch the app, click the Network Drives option on the navigation panel.

### Requirements

- eg. Fix crash that occurs when...
- eg. Correct action happens when clicking...

### Files Version

2.4.60.0

### Windows Version

10.0.22621.1485

**Comment by Josh65-2201** [comment]
Merging with #8304

In future please upload the logs file and don't copy and paste the contents in.

**Comment by yaira2** [comment]
This seems like a different issue, #8304 is for the sidebar whereas this is for the network folder.

**Comment by Josh65-2201** [comment]
They talk about two thing. They refer to the sidebar with

> The top half would be all of the PCs on my network

**Comment by jaseinatl** [comment]
The #8304 dicket has all of the network items inside the network folder in the sidepanel and is complaining about the extra step of opening the network folder. They want the contents of the network folder to show directly in the sidebar.

That is not my issue. I have NOTHING in my network folder so even if #8304 is resolved, my Network Sidepanel will show NOTHING. This is different than my network panel having a network folder with all the items in it.

**Comment by yaira2** [comment]
@jaseinatl can you clarify which issues apply here?

1. The sidebar isn't displaying the mapped drives
2. The network folder isn't displaying the mapped drivers
3. Both?

**Comment by Libroru** [comment]
Right now it bugs out the indexing algorithm... If I have no network drives in that folder and click it, I am unable to open any other folders until I close the app and open in it back up again.

**Comment by Josh65-2201** [comment]
Thats a known and different issue #7259

**Comment by jaseinatl** [comment]
@yair100 Sorry for the delay. Here's what I have found: 

If I leave the window overnight with it indexing my network, when I return the next day, it still shows nothing in the main panel.
However, if I then click the Network folder again, I get the busy cursor and in about 7-10 minutes, I see only the network computers, no drives. Which is a step in the right direction. None of the other network devices show up and no network drives (both locally mapped and shared or remotely mapped) show up.

So the answer is 3 unless I wait 24 hrs and click the network folder in the same window again. CPU usage is outta control and when this happened the first time, I just trained myself not to trust the Network Drives side panel. I had to restart my PC and at that point went to bed frustrated when it happened again on a fresh reboot. I decided to let it run overnight to see if it would cache the index at least by the next morning and it appears that it did not although it was about 23.8 hours shorter time when re-indexing. 

What would be ideal is a NETWORK folder and under the NETWORK folder (not Network Drives) would be three categories of icons: Devices (including smart wifi connected items, router, smart switches,), Computers (all networked computers) and Drives (all locally mapped drives that access network shares). This would put it miles ahead of Explorer.

Also of note: trying it again today, When clicking on the Network folder the item count at the bottom of the window says 4625 items (for about 5 minutes) then it says 9 items. The main window, however, shows 6 items in one section and 1 item in another. The strange thing is 2 items are missing for the total given, None of the network drives are showing and things like unmapped network devices like printer, router, jellyFin server, Plex server and Google Display are showing up. Then in a second section, Sonos Play5 is showing up. No network drives.  And without a section label, it's hard to say what it's thinking. It's still missing a lot of network devices and all of the network drives.

So the speed is improving, but the results are still bunk. Thanks for your time and help. Sorry to be so negative, I really love the software. Just trying to help.

**Comment by yaira2** [comment]
@jaseinatl we appreciate the feedback, it helps us know what areas need more improvement and having the issue on our tracking board helps us keep track of progress.

**Comment by jaseinatl** [comment]
Unfortunately, every time I tried to use your app to load a file through
any Adobe app, your app would hang and I'd have to force quit it and
return to the regular Windows explorer. In fact, I had to uninstall your
app and cannot use it on my system, so I am going to be of little help now.
I repeatedly locked up while it was running. It's a great looking app that
does exactly what I need, but it just doesn't work. Good luck with
development, I will check back later to see how you are doing.

Jase
